### ISSUE TITLE
lightgray may be too light

### ISSUE THREAD

**Issue by dlespiau** [weaveworks/ui-components#65]
We've seen a number of badly calibrated monitors that don't really show `lightgray` https://github.com/weaveworks/ui-components/blob/master/src/theme/weave.js#L18 well and make it as white as white.

We may want to make lightgray a bit more gray

**Comment by fbarl** [comment]
Since recently, @bia and I have been using http://snook.ca/technical/colour_contrast/colour.html for testing contrast compatibility between neighbouring colors - basically it tells us whether really bad screens still differentiate between two colors enough :)

**Comment by bia** [comment]
Yes, as a quantitative guide we've been only choosing color combinations which are WCAG 2 AA Compliant.
